ababber/fun-kickoff-project
# Tests for logout controller

### What this PR does (required):
- added logout controller tests and updated README with how to run back-end tests

### Any information needed to test this feature (required):
- go into the server directory `cd server`
- ensure your local .env file is setup correctly
- open the package.json file in the server directory
- in the package.json file, add your local .env file to the test script: "DOTENV_CONFIG_PATH=`your_local.env` mocha -r dotenv/config ./test/*.test.js"
- run `npm run test` to start the tests while in the server directory
- if the environment is setup correctly, the test for the logout controller will pass
